### ISSUE TITLE
Article Image Alt Tags Missing in contentarray.html.twig Joomla

### ISSUE THREAD

**Issue by DamirPecnik** [gantry/gantry5#2076]
The image Alt text for both the Intro Image and the Full Article Image for an article contentarray.html.twig is missing. Any way particle can pull that from the Joomla Article.
